_includes/urls

```diff
@@ -1,7 +1,40 @@
 {%- if page.version == "master" && include.component == "calicoctl" -%}
-https://projectcalico.org/builds/calicoctl
+{% assign version = site.data.archives.first | append: ".0" %}
 {%- elsif include.component == "calicoctl" -%}
-https://github.com/projectcalico/calicoctl/releases/download/{{site.data.versions.first.components.calicoctl.version}}/calicoctl
-{%- else -%}
-{% comment %} Invalid case {% endcomment %}
-{%- endif -%}
+{% assign version = site.data.versions.first.components.calicoctl.version %}
+{% endif %}
+{% tabs %}
+<label:Linux,active:true>
+<%
+```bash
+curl -O -L  "https://github.com/projectcalico/calicoctl/releases/download/{{ version }}/calicoctl"
```

Comment:
Why move the curl inside of the `urls` include? Do we use this include anywhere else?

Reply:
I am not sure why it is implemented this way, this file is only included once in `ctl-binary-install.md` which makes it a bit weird.

Reply:
I think it would be better to leave the `urls` file for now since it is being used two time in one file.